### DISCUSSION TITLE
How to exclude routes from build?

### DISCUSSION THREAD

**Discussion by webdeb** [vercel/next.js#65322] [Help] [answered]
### Summary

Here I've described an idea to build multiple apps out of one, for this I simple import service routes from the main app and build a separate service to serve those routes.
https://github.com/vercel/next.js/discussions/65303

Now, I basically need to exclude those routes from the build, or maybe even better to rewrite them, but as I understand those are proxied on the client? Don't get it really=)

Best solution would be to just exlude them from the build completely, and instead I would probably build router to map those routes to another service based on path somewhere in the infrastructure routing.

**Comment by icyJoseph** [comment, marked as answer]
Could you look into doing: https://github.com/vercel/next.js/discussions/51891

**Reply by webdeb** [reply]
Cool, I really like the solution, cause it keeps everything in one app 👍 
..going to test it :)
